src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -64,8 +64,8 @@ library GPv2Encoding {
     /// @dev A struct representing arbitrary contract interactions.
     /// Submitted to [`GPv2Settlement.settle`] for code execution.
     struct Interaction {
-        bytes callData;
         address target;
+        bytes callData;
```

Comment:
Whoops! I didn' realize I had done that. Is it safe to assume that the ordering is alphabetical by type?

Reply:
I think this makes more sense given that the encoding is `address || callDataLength || callData` (so the calldata comes after). The other point I see is that you call it like `address(target).call(callData)` so this proposed ordering feels quite natural. I hope it wasn't chosen to be ordered alphabetically :see_no_evil:

Reply:
It was perfectly fine as it was before, the reason I changed it is that I didn't want to add a small extra complication to the assembly code. If the two had been left like they were, I would have needed to add a comment explaining that a `bytes` vector is stored in a struct as a pointer, and then that I was skipping 32 bytes to write to the memory location of `target`. Instead, now I write immediately to the memory location of `target` and it doesn't really matter how `callData` is laid down in memory.